public/about.html

```diff
@@ -123,12 +126,12 @@ <h1 class="page-title hyphenate" id="lead">About BLE LTER</h1>
       <hr>
       <br>
       <h2 id="policies">Policies</h2>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/BLE-DEI-Plan">Diversity, Equity, and Inclusion Plan</a></button>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/ble-authorship-policy">Authorship Policy</a></button>
-      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/ble-data-policy">Data Policy</a></button>
+      <button class="btn btn-lg btn-outline-primary m-1"><a rel="external noopener" target="_blank" href="https://utexas.box.com/v/BLE-DEI-Plan"  aria-label="open diversity equity inclusion plan in new tab">Diversity, Equity, and Inclusion Plan</a></button>
```

Comment:
why is the aria-label needed since the link has text?

Reply:
![image](https://user-images.githubusercontent.com/42045942/116716055-ea8c7d00-a99c-11eb-8f32-862b87334c27.png)